internal/serverless/project.go

```diff
@@ -196,6 +250,11 @@ func (p *Project) CreateAgentPolicy(stackVersion string, kibanaClient *kibana.Cl
 		Namespace:         "default",
 		MonitoringEnabled: []string{"logs", "metrics"},
 	}
+
+	if logstashEnabled {
+		policy.DataOutputID = FLEET_LOGSTASH_OUTPUT
+	}
```

Comment:
Nit. It looks a bit weird to me that this method needs to know about logstash, wdyt about passing instead the output ID directly?
```suggestion
		DataOutputID: dataOutputID,
	}
```
And in the caller:
```
		dataOutputID = ""
		if settings.LogstashEnabled {
			dataOutputID: FLEET_LOGSTASH_OUTPUT
		}
		err = project.CreateAgentPolicy(options.StackVersion, sp.kibanaClient, dataOutputID)
```

Reply:
The agent policy creation complains if the `dataOutputId` does not exist. Hence the output is created first and then referred in the agent policy.